schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -44,7 +44,13 @@
                   { "$ref": "http://schema.management.azure.com/schemas/2015-04-08/Microsoft.DocumentDB.json#/resourceDefinitions/databaseAccounts" },
                   { "$ref": "http://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/resourceDefinitions/secrets" },
                   { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs" },
-                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/virtualMachines" },
+                  { "$ref": "http://schema.management.azure.com/schemas/2016-05-15/Microsoft.DevTestLab.json#/resourceDefinitions/labs_artifactsources" },

```

Comment:
Please add these new resource types to 2014-04-01-preview/deploymentTemplate.json too.